public/js/positionForm.js

```diff
@@ -0,0 +1,102 @@
+window.onload = () => {
+  const navButton = document.getElementById('positionsNav');
+  navButton.classList.add('activePage');
+
+  const params = new URLSearchParams(window.location.search);
+  const positionId = params.get('_id');
+
+  const clientInput = document.getElementById('client');
+  const jobDescriptionInput = document.getElementById('jobDescription');
+  const vacancyInput = document.getElementById('vacancy');
+  const professionalProfilesInput = document.getElementById('professionalProfiles');
+  const isOpenInput = document.getElementById('isOpen');
+
+  const form = document.getElementById('form');
+  const saveButton = document.getElementById('saveButton');
+  const errorMessage = document.getElementById('error_msg');
+
+  saveButton.disable = !!params.get('_id');
```

Comment:
use `positionId` instead of invoke `params.get('_id')` again

Reply:
done